api/src/main/java/io/strimzi/api/kafka/model/KafkaClusterSpec.java

```diff
@@ -71,7 +71,8 @@
     private Probe livenessProbe;
     private Probe readinessProbe;
     private JvmOptions jvmOptions;
-    private Map<String, Object> metrics;
+    private KafkaJmxRemote jmxRemote;
```

Comment:
I was wondering that as we have `jvmOptions` isn't it better `jmxOptions` because it's really about JMX options, or?

Reply:
Yea we can do that, I just wanted something different from `jmxMetrics` as there is already a `metrics` field

Reply:
Totally agree, for this reason I was suggesting something like `jmxOptions` :-)

Reply:
Can do, just wanted something different from `metrics`. I agree that it should be `jmxOptions` to follow naming conventions and it fits with the jmxTrans deployment later on.